doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -236,6 +236,7 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
        uses ``/usr/share/acrn/bios/VSBL.bin`` as the vSBL image
 
    * - :kbd:`--ovmf [w,]<ovmf_file_path>`
+   * - :kbd:`--ovmf [w,]code=<ovmf_code_file>,vars=<ovmf_vars_file>`
```

Comment:
```suggestion
       :kbd:`--ovmf [w,]code=<ovmf_code_file>,vars=<ovmf_vars_file>`
```

I think this will solve your `make doc` failure @peterfang